test/e2e_new/features/broker_deleted_recreated.go

```diff
@@ -82,3 +83,24 @@ func BrokerConfigMapDoesNotExist() *feature.Feature {
 
 	return f
 }
+
+// BrokerCannotReachKafkaCluster tests that a broker can be deleted even when KafkaCluster is unreachable.
+func BrokerCannotReachKafkaCluster() *feature.Feature {
+	f := feature.NewFeatureNamed("delete broker ConfigMap first")
```

Comment:
copy-pasted from above?
```suggestion
	f := feature.NewFeatureNamed("delete broker with unreachable Kafka cluster")
```

Reply:
LOL my bad